Gemfile.lock

```diff
@@ -36,4 +37,4 @@ DEPENDENCIES
   rspec
 
 BUNDLED WITH
-   1.14.4
+   1.14.6
```

Comment:
Shouldn't need to update the Gemfile.lock. Let us know if there's an issue with your setup where you're having to do this frequently.